### DISCUSSION TITLE
[Lesson 4] cannot import name 'networks' from 'brownie'

### DISCUSSION THREAD

**Discussion by DarrenWayn** [smartcontractkit/full-blockchain-solidity-course-py#682] [Q&A]
<img width="874" alt="Screen Shot 2022-01-03 at 18 03 01" src="https://user-images.githubusercontent.com/22322944/147923527-84287ad0-0e0b-4c2d-b011-6b304757b5d6.png">

I have this problem when brownie when "run scripts/deploy.py -- networks rinkeby"

anybody knows how to fix this?

**Comment by cromewar** [comment]
Hello @DarrenWayn  you have a space between `--` and `network` which is incorrect, you should run the command with `--network` and also on singular, not networks.

I hope this info might help.
